### PR TITLE
added pluralsight course

### PR DESCRIPTION
**Pull request by johnpapa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -316,6 +316,7 @@
 - [Real-time Grid Component Laravel, Vue.js, Vuex & Socket.io](https://www.youtube.com/watch?v=Jxefsv5Zqkw&t=3s)
 - [VueJS 2 - The Complete Guide (incl. Vuex) - Udemy Tutorial](https://www.udemy.com/vuejs-2-the-complete-guide)
 - [Develop Web Apps with Vue.js](https://egghead.io/courses/develop-web-apps-with-vue-js) on [egghead.io](https://egghead.io/)
+- [Vue: Getting Started](https://www.pluralsight.com/courses/vue-getting-started) on [Pluralsight](https://www.pluralsight.com)
```
Comment (Scrum) [inline README.md]: @johnpapa Needs to be at the end of the list.

Comment (johnpapa) [reply README.md]: What do you need me to update?

Comment (Scrum) [reply README.md]: You need to move the record to the end according to the rules https://github.com/vuejs/awesome-vue/blob/master/CONTRIBUTING.md#contributing-guide

Comment (johnpapa) [reply README.md]: ok. moved to bottom

**Pull request merged**